src/free-style.ts

```diff
@@ -170,10 +170,11 @@ function stringifyProperties (properties: Properties) {
 
   for (const [name, value] of properties) {
     if (value != null) {
-      if (Array.isArray(value)) {
-        result.push(value.filter(x => x != null).map(x => styleToString(name, x)).join(';'))
+      const value2 = typeof value === 'function' ? value() : value
```

Comment:
PostCSS, vendor prefix, things that expect numbers or strings. All modules that can currently augment a style object would need to be notified that this change in expectations has occurred, as it won't be very useful for a style processor to skip a function instance because it doesn't understand it (or worse, crash or omit it altogether thinking it's invalid) and, as a result, not update the style with vendor prefixes.